### PR TITLE
Add `.scwrc` file format (fixes #121)

### LINKED ISSUE #121: Describe format of .scwrc
#111 added support for `.scwrc` and made it the "recommended" authentication method if no credentials are supplied the other way.
```
✗ terraform plan

Error: Error asking for user input: 1 error(s) occurred:

* provider.scaleway: Error configuring scaleway: organization: error loading default: error loading default: open /home/anatoli/.scwrc: no such file or directory
```
I don't mind setting this file, but could anyone describe the format, please?
```
✗ terraform -v
Terraform v0.11.13
+ provider.scaleway v1.9.2
```

### PR DESCRIPTION
**Pull request by abitrolly**
Used `scw` tool to generate one

### CODE REVIEW

**Comment by nicolai86** [comment]
Given the file is generated by the scw utility do you think it’s useful to copy the structure here, or would the link to it be sufficient?

**Comment by QuentinBrosse** [comment]
We are working on a new version of the Scaleway developer tools configuration. The path and the format of the file will change, so keep in mind that this documentation will change in the next weeks.

Note that this new format will be backward compatible and will handle the migration from the old format to the new one.

**Comment by abitrolly** [comment]
That will break Terraform setup for people. At least the provider should be
able to understand new format and it would be nice to get some details
about it beforehand.

**Comment by QuentinBrosse** [comment]
Hi @abitrolly,

We are taking the backward compatibility issues very seriously. Therefore, the configuration update will not be breaking change. You will find all the details on Step 1.3 of [the version 2 roadmap](https://github.com/terraform-providers/terraform-provider-scaleway/issues/125) and in a dedicated issue. 🙂

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -108,6 +108,9 @@ provider "scaleway" {
 - **SCALEWAY_REGION**: The Scaleway region
 
 Alternatively the `~/.scwrc` will be parsed to load your Scaleway CLI configuration, if it exists.
+The format of the config file is JSON string `{"organization":"","token":"","version":"v1.19"}`
+where `version` is the version of [scw](https://github.com/scaleway/scaleway-cli) tool.
+
```
Comment (huseyin) [inline website/docs/index.html.markdown]: there is an empty line here

**Comment by abitrolly** [comment]
I updated PR. There are no references to `config.yaml` in `scaleway-cli` repository. Therefore I think it is still actual.

https://github.com/scaleway/scaleway-cli/search?q=config.yaml&unscoped_q=config.yaml

**Comment by abitrolly** [comment]
Filled here https://github.com/scaleway/scaleway-cli/issues/558

**Comment by QuentinBrosse** [comment]
The [new CLI](https://github.com/scaleway/scaleway-cli/issues/558#issuecomment-517202416) and the new Terraform provider (that will release in September too) will share the same configuration file.

**Comment by abitrolly** [comment]
I think I am hitting https://github.com/terraform-providers/terraform-provider-scaleway/issues/95 with the old version.

Does not seem that 1.11 improves image management. The work with list of images through authenticated API is surely confusing https://community.scaleway.com/t/list-of-image-uuids-without-api/7672/3 

I am not happy with Scaleway, because I am paying for instances that are not functioning for over half of the year, because I don't have the power to troubleshoot them. https://community.scaleway.com/t/arm64-hangs-with-ubuntu-18-04-fully-updated/7681/4 Only after this issue is resolved, or made into Ubuntu bug report, I will be able to run anything with Terraform.

**Comment by QuentinBrosse** [comment]
Hi @abitrolly,

I'm sorry to hear that. We have totally conscience that we have to improve the image management user experience both in Terraform and in our other developer tools. We will recode all our Terraform data sources on the next version (`v1.12`). Check out the [v2 roadmap](https://github.com/terraform-providers/terraform-provider-scaleway/pull/125) for more information. Other developer tools will also be improved too.

I have just created an issue related to the recode of data sources [here](https://github.com/terraform-providers/terraform-provider-scaleway/pull/225). Feel free to give some feedback in this one in order to keep the conversation focused on the `.scwrc` file here. :)

Note that you will have an answer on Scaleway community today, sorry for the delay.

**Comment by abitrolly** [comment]
@QuentinBrosse do you have plan for better image management in upcoming version? If not, I think I can join you as an interested party for a couple of weeks to champion improvements to this flow.

**Comment by QuentinBrosse** [comment]
The [v1.11.0](https://github.com/terraform-providers/terraform-provider-scaleway/blob/master/CHANGELOG.md#1110-october-01-2019) introduces a new configuration mechanism. Therefore, `.scwrc` is now deprecated in favor of a new YAML shared configuration file.

Here is [the documentation](https://www.terraform.io/docs/providers/scaleway/index.html#shared-configuration-file) of this new configuration file. You can find more details [in the documentation of our new Go SDK](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md#scaleway-config).

As it is a documentation enhancement on a deprecated configuration file, I will close this pull request. Feel free to open a new issue if you need better documentation of the [provider authentication](https://www.terraform.io/docs/providers/scaleway/index.html#authentication).

As a reminder, you can find the v2 roadmap [here](https://github.com/terraform-providers/terraform-provider-scaleway/issues/125).

Thanks a lot for your contribution guys, you rocks! 👌

**Pull request closed without merging**